policies/branch-policy.md

```diff
@@ -0,0 +1,100 @@
+Branch Policy
+=============
+
+The openssl repository contains the following maintained branches:
+
+## The master branch
+
+- Any type (bug fix, feature, refactoring…) of pull requests is allowed.
+
+- The development of the next minor or major release happens there.
+  Which kind of release it is is defined by OMC at some point after the
+  master branch is opened after the previous release was branched.
+  Implicitly it means that any API/ABI breaking changes are not allowed
+  until OMC defines the intended kind (minor/major) of the next release.
+
+**If the next minor branch is approved the following applies.**
+
+- The development of the next major release happens there. Implicitly it
```

Comment:
Do you really mean major here?

Reply:
Yes, we are talking here about what is the purpose of the master branch.